src/app/hooks/useValidatorRegistrationFlow.ts

```diff
@@ -71,12 +71,20 @@ const BULK_MODE_TO_ROUTES: NavigationRoutes = {
       [EValidatorFlowAction.FIRST_REGISTER]: config.routes.SSV.VALIDATOR.FUNDING_PERIOD_PAGE,
       [EValidatorFlowAction.SECOND_REGISTER]: config.routes.SSV.MY_ACCOUNT.CLUSTER.ADD_VALIDATOR,
     },
-    [config.routes.SSV.MY_ACCOUNT.CLUSTER.ADD_VALIDATOR]:  config.routes.SSV.VALIDATOR.ACCOUNT_BALANCE_AND_FEE,
+    [config.routes.SSV.MY_ACCOUNT.CLUSTER.ADD_VALIDATOR]: config.routes.SSV.VALIDATOR.ACCOUNT_BALANCE_AND_FEE,
     [config.routes.SSV.VALIDATOR.DISTRIBUTION_METHOD.DISTRIBUTE_SUMMARY]: config.routes.SSV.MY_ACCOUNT.CLUSTER.UPLOAD_KEYSHARES,
   },
   [EBulkMode.MULTI]: {
     [config.routes.SSV.MY_ACCOUNT.CLUSTER.ROOT]: config.routes.SSV.MY_ACCOUNT.CLUSTER.DISTRIBUTION_METHOD_START,
-    [config.routes.SSV.MY_ACCOUNT.CLUSTER_DASHBOARD]: { [EValidatorFlowAction.ADD_CLUSTER]: config.routes.SSV.VALIDATOR.HOME },
+    [config.routes.SSV.MY_ACCOUNT.CLUSTER.DISTRIBUTION_METHOD_START]: {
+      [EValidatorFlowAction.GENERATE_KEY_SHARES_ONLINE]: config.routes.SSV.MY_ACCOUNT.CLUSTER.UPLOAD_KEY_STORE,
+      [EValidatorFlowAction.GENERATE_KEY_SHARES_OFFLINE]: config.routes.SSV.MY_ACCOUNT.CLUSTER.DISTRIBUTE_OFFLINE,
+      [EValidatorFlowAction.SECOND_REGISTER]: config.routes.SSV.MY_ACCOUNT.CLUSTER.UPLOAD_KEYSHARES,
+
+    },
+    [config.routes.SSV.MY_ACCOUNT.CLUSTER_DASHBOARD]: {
+      [EValidatorFlowAction.ADD_CLUSTER]: config.routes.SSV.VALIDATOR.HOME,
```

Comment:
If there is only one option you can map route to route directly without an action. Action are only needed when there might be different navigations fro the current route/location.

Reply:
changed in another PR

Reply:
ok, please make sure to add it before merging to stage.